Art/niyati/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="index.css" type="text/css" />
+  <title>Assignment 10</title>
+</head>
+
+<body>
+  <div class="container">
+    <div class="wrapper">
+      <div class="city"></div>
+      <div class="car">
+        <img src="./image/car.png" alt="" />
+        <div class="wheel back">
+          <img src="./image/wheel3.png" alt="" />
+        </div>
+        <div class="wheel front">
+          <img src="./image/wheel3.png" alt="" />
+        </div>
+      </div>
```

Comment:
## Missing files
Missing files ( and folder image) in your folder:

- image/car.png
- image/wheel3.png